services/market/shared/market/sources/exchangeratesapi.js

```diff
@@ -0,0 +1,107 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2021 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const { HTTP, Logger, CacheRedis } = require('lisk-service-framework');
+
+const BluebirdPromise = require('bluebird');
+
+const requestLib = HTTP.request;
+const logger = Logger();
+
+const config = require('../../../config.js');
+
+const exchangeratesapiCache = CacheRedis('exchangeratesapi_prices', config.endpoints.redis);
+
+const apiEndpoint = config.endpoints.exchangeratesapi;
+const accessKey = config.access_key.exchangeratesapi;
+
+const currencies = ['EUR', 'USD', 'CHF', 'BTC'];
+
+const symbolMap = {
+    EUR_USD: 'EURUSD',
+    EUR_CHF: 'EURCHF',
+    EUR_BTC: 'EURBTC',
+};
+
+const fetchAllMarketTickers = async () => {
+    try {
+        const allMarketTickers = {};
+        await BluebirdPromise.all(
+            currencies.map(async (baseCurrency) => {
+                const response = await requestLib(`${apiEndpoint}/latest?access_key=${accessKey}&base=${baseCurrency}&symbols=${currencies.join(',')}`);
+                if (response) allMarketTickers[baseCurrency] = response.data.rates;
+            }),
+        );
+        return allMarketTickers;
```

Comment:
As a good to have, please consider removing the `baseCurrency` from the `symbols` list as the relationship would always be a constant equal to `1`.

Also, please consider renaming the methods/variables to be more appropriate and reflective of their purpose.

```suggestion
const fetchAllCurrencyConversionRates = async () => {
    try {
        const allMarketConversionRates = {};
        await BluebirdPromise.all(
            currencies.map(async (baseCurrency) => {
                const remainingCurrencies = currencies.filter(c => c !== baseCurrency);
                const response = await requestLib(`${apiEndpoint}/latest?access_key=${accessKey}&base=${baseCurrency}&symbols=${remainingCurrencies.join(',')}`);
                if (response) allMarketConversionRates[baseCurrency] = response.data.rates;
            }),
        );
        return allMarketConversionRates;
```